modules/lang-painless/build.gradle

```diff
@@ -84,8 +84,10 @@ shadowJar {
 
 tasks.validateNebulaPom.dependsOn tasks.generatePomFileForShadowPublication
 tasks.validateShadowPom.dependsOn tasks.generatePomFileForNebulaPublication
-tasks.publishNebulaPublicationToMavenLocal.dependsOn tasks.generatePomFileForShadowPublication
-tasks.publishShadowPublicationToMavenLocal.dependsOn tasks.generatePomFileForNebulaPublication
+tasks.findAll { it.name.startsWith "publishNebulaPublication" }
```

Comment:
I think we could simply that with

```
tasks.withType(AbstractPublishToMaven)*.dependsOn "generatePomFileForShadowPublication", "generatePomFileForNebulaPublication"
```

both Nebula and Shadow are Maven-based publications

Reply:
Agreed